.github/workflows/merge-request.yml

```diff
@@ -47,6 +47,50 @@ jobs:
       - name: Upload coverage reports to Codecov
         uses: codecov/codecov-action@v3
 
+  eas-web:
+    name: EAS Web
+    needs: code-quality
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Check for EXPO_TOKEN
+        run: |
+          if [ -z "${{ secrets.EXPO_TOKEN }}" ]; then
+            echo "You must provide an EXPO_TOKEN secret linked to this project's Expo account in this repo's secrets. Learn more: https://docs.expo.dev/eas-web/github-actions"
+            exit 1
+          fi
+
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 22.x
+          cache: yarn
+
+      - name: Setup EAS
+        uses: expo/expo-github-action@v8
+        with:
+          eas-version: latest
+          token: ${{ secrets.EXPO_TOKEN }}
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Build web app
+        working-directory: packages/app
+        run: npx expo export --platform web
+
+      - name: Deploy web app
+        id: deploy-web
+        working-directory: packages/app
+        run: |
+          result="$(eas deploy --non-interactive --json)"
+          echo "$result"
+          echo "deployment=$result" >> $GITHUB_OUTPUT
```

Comment:
Can you somehow make this url be accessible from the pr? Dies GitHub have environments with urls like gitlab? Or how to make job output accessible from the pr page for ease of testing?